src/js/crypto.cpp

```diff
@@ -606,6 +608,8 @@ namespace ccf::js
           {wrapping_key, wrapping_key + wrapping_key_size},
           {key, key + key_size});
 
+        OPENSSL_cleanse(wrapping_key, wrapping_key_size);
```

Comment:
Here this is a private key, but we still should not scrub the user input, instead it's the temporary object we should scrub (line 608).